.github/workflows/main.yml

```diff
@@ -92,3 +94,38 @@ jobs:
         uses: coverallsapp/github-action@master
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+  build-docker-images:
+    if: ${{ github.ref_type == 'tag' || github.event.inputs.build_docker_images }}
+    runs-on: ubuntu-20.04
```

Comment:
Use `ubuntu-latest` so we don't have to update it when a new Ubuntu release comes out.

Reply:
I used to have it set to `ubuntu-latest` but when it updated, it broke our builds. I'd rather have explicit control over updating so that we can do it when its convenient to us rather than having to put out fires. See #50 and #51.